pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -83,8 +147,21 @@ class LightningKokkos(LightningQubit):
         "Identity",
     }
 
-    def __init__(self, wires, *, sync=True, c_dtype=np.complex128, shots=None, batch_obs=False):
-        super().__init__(wires, c_dtype=c_dtype, shots=shots)
+    def __init__(
+        self, wires, *, sync=True, c_dtype=np.complex128, shots=None, batch_obs=False, analytic=None
```

Comment:
```suggestion
        self, wires, *, sync=True, c_dtype=np.complex128, shots=None, batch_obs=False
```

Analytic isn't an option we use. I'd suggest mapping this closer to the LGPU class structure. We also do not make use of batch_obs yet, but as we intend to improve performance with this later we can keep for now.